readme.md

```diff
@@ -660,6 +660,30 @@ As of version `5.0.0` it uses source maps to report coverage for your actual cod
 
 ## FAQ
 
+### How do I test HTTP endpoints?
+
+AVA doesn't have an official assertion library for endpoints, but a great option is [`supertest-as-promised`](https://www.npmjs.com/package/supertest-as-promised).
+Assuming you are using express, you might set up a server for each test:
+```
+let app;
+test.beforeEach(() => {
```

Comment:
ahhhh, that kinda clears it up. I was thinking that `beforeEach` was guaranteed to run immediately before the test *on that thread*. So is `beforeEach` guaranteed to run on the same thread as the test?

PS, I like your 3rd option of creating the app, I'll update the readme to reflect it